packages/cadl-vs/Microsoft.Cadl.VS.sln

```diff
@@ -1,8 +1,11 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31202.260
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31919.166
 MinimumVisualStudioVersion = 16.0.31202.260
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Cadl.VS2019", "VS2019\Microsoft.Cadl.VS2019.csproj", "{3B3CF7E4-5B54-47EA-8BE3-60D8F806C69E}"
+	ProjectSection(ProjectDependencies) = postProject
```

Comment:
Add a dependency of VS2019 dependsOn VS2022 project that way when you build/F5 VS2019 to debug it also builds and deploys to VS2022

Reply:
Aha! Thanks

Reply:
Feeling silly now since I worked on SDK and project system and should have realized my mistake.